analysis/topEFT/work_queue_run.py

```diff
@@ -166,7 +166,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
 
 executor_args = {
     'master_name': '{}-workqueue-coffea'.format(os.environ['USER']),
-
+     
```

Comment:
This is a very minor comment, but it'd be good to not add this extraneous white space.

Reply:
I will get rid of that.